### ISSUE TITLE
Potential Issues with Hardcoded EMA Warmup Factor

### ISSUE THREAD

**Issue by kimbayashanologatiniburgettoilalalourvr** [ultralytics/yolov5#4959]
Hi there,

I wanted to report and also solicit some thoughts on a potential bug/hardcoded value for the EMA function in torch_utils.py

We noticed when working with this section of code that the time for the EMA to warmup is hardcoded to `2000` and not really accessible in any way other than directly to 'hack' in the new value. I think we've found that this might impact smaller runs, runs much shorter than the 10k iterations to get to 99% of the original config value.

This may work for COCO-only, but are there any thoughts on potentially externalizing this to a config file at some point? This might come in handy for users finetuning networks from previously-initialized model weights (with the model_iters stripped out from them &etc).

Here's a link to the original code:

https://github.com/ultralytics/yolov5/blob/2993c3fa7af7a76dd82349e3cf85e35e4254576b/utils/torch_utils.py#L322

**Comment by glenn-jocher** [comment]
@kimbayashanologatiniburgettoilalalourvr hi thanks for the review! EMA warmup is something I implemented myself for YOLOv5, it is not part of Ross Wightman's EMA module. 

The intention there is to allow for better results in early training, otherwise initial results will suffer very slow ramps. It's conceivable that parameter might be well placed in the hyp.yaml file alongside the other warmup parameters here. If you'd like to submit a PR for this or another solution please feel free, thanks!

https://github.com/ultralytics/yolov5/blob/070af88108e5675358fd783aae9d91e927717322/data/hyps/hyp.scratch.yaml#L10-L12
